libs/partners/zenguard/langchain_zenguard/tools.py

```diff
@@ -0,0 +1,87 @@
+import os
+from enum import Enum
+from typing import Any, Dict, List, Optional
+
+import requests
+from langchain_core.pydantic_v1 import BaseModel, Field, ValidationError, validator
+from langchain_core.tools import BaseTool
+
+
+class Detector(str, Enum):
+    ALLOWED_TOPICS = "allowed_subjects"
+    BANNED_TOPICS = "banned_subjects"
+    PROMPT_INJECTION = "prompt_injection"
+    KEYWORDS = "keywords"
+    PII = "pii"
+    SECRETS = "secrets"
+    TOXICITY = "toxicity"
+
+class DetectorAPI(str, Enum):
+    ALLOWED_TOPICS = "v1/detect/topics/allowed"
+    BANNED_TOPICS = "v1/detect/topics/banned"
+    PROMPT_INJECTION = "v1/detect/prompt_injection"
+    KEYWORDS = "v1/detect/keywords"
+    PII = "v1/detect/pii"
+    SECRETS = "v1/detect/secrets"
+    TOXICITY = "v1/detect/toxicity"
+
+class ZenGuardInput(BaseModel):
```

Comment:
why we need ZenGuardInput class

Reply:
this class shows how arguments expected our tool when call methods run, invoke, batch

Example 
```
zenguard_tool = ZenguardTool()
print(zenguard_tool.args) # >> {'prompts': {'title': 'Prompts', 'type': 'list'}, 'detectors': ...}
```